### PR TITLE
fix: update types for default slotted content

### PR DESCRIPTION
**Pull request by chrisdholt**
# Pull Request

## 📖 Description
This PR updates the return type for the `dangerousHTML()` helper to `DangerousHTMLDirective` and updates the typings in FAST Foundation usage for default slotted content options.

### 🎫 Issues
n/a

### CODE REVIEW

**Review by eljefe223 — APPROVED** [review]

File: packages/web-components/fast-foundation/src/combobox/stories/combobox.register.ts [hunk 1]
```diff
@@ -205,7 +206,7 @@ const styles = css`
 FASTCombobox.define({
     name: "fast-combobox",
     template: comboboxTemplate({
-        indicator: /* html */ `
+        indicator: /* html */ html`
```
Comment (janechu) [inline packages/web-components/fast-foundation/src/combobox/stories/combobox.register.ts:209]:
```suggestion
        indicator: html`
```

File: packages/web-components/fast-foundation/src/flipper/stories/flipper.register.ts [hunk 2]
```diff
@@ -85,14 +86,14 @@ const styles = css`
 FASTFlipper.define({
     name: "fast-flipper",
     template: flipperTemplate({
-        next: /* html */ `
+        next: /* html */ html`
```
Comment (janechu) [inline packages/web-components/fast-foundation/src/flipper/stories/flipper.register.ts:89]:
```suggestion
        next: html`
```

File: packages/web-components/fast-foundation/src/flipper/stories/flipper.register.ts [hunk 3]
```diff
@@ -85,14 +86,14 @@ const styles = css`
...
             <svg viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg">
                 <path
                     d="M4.023 15.273L11.29 8 4.023.727l.704-.704L12.71 8l-7.984 7.977-.704-.704z"
                 />
             </svg>
         `,
-        previous: /* html */ `
+        previous: /* html */ html`
```
Comment (janechu) [inline packages/web-components/fast-foundation/src/flipper/stories/flipper.register.ts:96]:
```suggestion
        previous: html`
```

File: packages/web-components/fast-foundation/src/menu-item/stories/menu-item.register.ts [hunk 4]
```diff
@@ -201,27 +202,33 @@ export const styles = css`
...
-                <path fill-rule="evenodd" clip-rule="evenodd" d="m8.1 12.7 7.1-8.2 1.6 1.4-8.6 9.9-4.4-4.5 1.5-1.5L8 12.7Z"/>
+                <path
+                    fill-rule="evenodd"
+                    clip-rule="evenodd"
+                    d="m8.1 12.7 7.1-8.2 1.6 1.4-8.6 9.9-4.4-4.5 1.5-1.5L8 12.7Z"
+                />
             </svg>
         `,
-        expandCollapseGlyph: /* html */ `
+        expandCollapseGlyph: /* html */ html`
```
Comment (janechu) [inline packages/web-components/fast-foundation/src/menu-item/stories/menu-item.register.ts:219]:
```suggestion
        expandCollapseGlyph: html`
```

File: packages/web-components/fast-foundation/src/progress-ring/stories/progress-ring.register.ts [hunk 5]
```diff
@@ -72,7 +73,7 @@ const styles = css`
 FASTProgressRing.define({
     name: "fast-progress-ring",
     template: progressRingTemplate({
-        indeterminateIndicator: /* html */ `
+        indeterminateIndicator: /* html */ html`
```
Comment (janechu) [inline packages/web-components/fast-foundation/src/progress-ring/stories/progress-ring.register.ts:76]:
```suggestion
        indeterminateIndicator: html`
```

**Review by janechu — APPROVED** [review]
Looks good, correct me if I'm wrong but I think we can remove the html comments since it actually returns an `html` tag template literal now. I would only hold on the prerelease changelog.

**Pull request merged**